File: KuantumJava/AntiMadde.java
// çok tehlikeli nesne (IKritik uygulanır)
public class AntiMadde extends KuantumNesnesi implements IKritik {
    public AntiMadde(String id) {
        super(id);
    }
    @Override
    public void analizEt() throws KuantumCokusuException {
        setStabilite(getStabilite() - 25);
        System.out.println("Evrenin dokusu titriyor...");
    }
    @Override
    public void acilDurumSogutmasi() {
        try {
            setStabilite(getStabilite() + 50);
            System.out.println("Acil durum soğutması yapıldı, stabilite yenilendi.");
        } catch (KuantumCokusuException e) {
            // 100'den fazla olmaz
        }
    }
}

File: KuantumCSharp/KuantumOdev/AntiMadde.cs
using System;

// çok tehlikeli nesne (IKritik uygulanır)
public class AntiMadde : KuantumNesnesi, IKritik
{
    public AntiMadde(string id)
    {
        ID = id;
        Stabilite = 70.0; // örnek başlangıç stabilitesi
        TehlikeSeviyesi = new Random().Next(1, 11);
    }

    public override void AnalizEt()
    {
        Stabilite -= 25;
        Console.WriteLine("Evrenin dokusu titriyor..."); 
    }

    public void AcilDurumSogutmasi()
    {
        Stabilite += 50;
        Console.WriteLine("Acil durum soğutması yapıldı, stabilite yenilendi.");
    }
}
